### ISSUE TITLE
User Feedback: Add Target attribute to open the link in a new window

### ISSUE THREAD

**Issue by Jaspreet099** [moja-global/community-website#332]
### Describe what you are trying to do and why it is not optimal.

![image](https://user-images.githubusercontent.com/75212002/172177900-adffcdcf-d07a-460a-9fb6-5a2d73b5f6d7.png)

Add Target _blank in the footer section(Follow Us) to open links in a new tab.

**Comment by padmajabhol** [comment]
@ayush78781 next time, please let us know before opening a PR. Many times, people who open a particular issue might also be interested to come up with a solution for it. There might also be a situation where we might not need to solve an issue. Prior notifications can help both contributors and maintainers 😅

**Comment by ayush78781** [comment]
> @ayush78781 next time, please let us know before opening a PR. Many times, people who open a particular issue might also be interested to come up with a solution for it. There might also be a situation where we might not need to solve an issue. Prior notifications can help both contributors and maintainers 😅

Hey Padmaja! I completely understand that. Actually what happened was, I already worked on this issue considering it as an enhancement without even looking at this particular issue. When I saw it open, I linked that pr to it. I will make sure from the next time not to repeat so. Sorry for the inconvenience caused.

**Comment by arishmaurya** [comment]
i want to work on this issue @ayush78781 ? And before two comments you have merged some changes so is this issue is still open ?

**Comment by ayush78781** [comment]
> i want to work on this issue @ayush78781 ? And before two comments you have merged some changes so is this issue is still open ?

No, its not open now. You can work on #334 if you would like to.
